Makefile

```diff
@@ -0,0 +1,29 @@
+
+TARGET_OS=linux
+SHELL_GET_TARGET_ARCH := $(shell test ! -z $(TARGET_ARCH) && echo $(TARGET_ARCH) || \
+	arch \
+	| sed s/x86_64/amd64/g \
+	| sed s/i386/amd64/g \
+	| sed s/aarch64/arm64/g \
+)
+TARGET_TAG=$(TARGET_OS)-$(SHELL_GET_TARGET_ARCH)
+CONTAINER_NAME=labop
+TAGGED_NAME=$(CONTAINER_NAME):$(TARGET_TAG)
+REGISTRY_ROOT=danbryce
+
+docker:
+	DOCKER_BUILDKIT=1 docker buildx build \
+		--output "type=docker" \
+		--platform $(TARGET_OS)/$(SHELL_GET_TARGET_ARCH) \
+		-t ${TAGGED_NAME} -f ./Dockerfile .
+	docker tag $(TAGGED_NAME) $(REGISTRY_ROOT)/$(TAGGED_NAME)
+	docker push $(REGISTRY_ROOT)/$(TAGGED_NAME)
```

Comment:
Will there be any permission issues caused when a user tries to push to your Docker registry without credentials?

Reply:
modified to use ghcr.io/bioprotocols/labop